baseplate/events/queue.py

```diff
@@ -38,33 +58,28 @@ def __init__(self, topic, event_type, timestamp=None, id=None):
             self.timestamp = time.time() * 1000
         self.id = id or uuid.uuid4()
         self.payload = {}
+        self.payload_types = {}
 
-    def get_field(self, key, obfuscated=False):
+    def get_field(self, key):
         """Get the value of a field in the event.
 
         If the field is not present, :py:data:`None` is returned.
 
         :param str key: The name of the field.
-        :param bool obfuscated: Whether to look for the field in the obfuscated
-            payload.
 
         """
 
-        if not obfuscated:
-            payload = self.payload
-        else:
-            payload = self.payload.get("obfuscated_data", {})
-        return payload.get(key, None)
+        return self.payload.get(key, None)
 
-    def set_field(self, key, value, obfuscate=False):
+    def set_field(self, key, value, obfuscate=False, kind=FieldKind.NORMAL):
         """Set the value for a field in the event.
 
         :param str key: The name of the field.
         :param value: The value to set the field to. Should be JSON
             serializable.
-        :param bool obfuscate: Whether or not to put the field in the obfuscated
-            section. This is used for sensitive info like IP addresses that must
-            be treated with care.
+        :param :py:class:`FieldKind` kind: The kind the field is. Used
```

Comment:
the :py:class: is unnecessary here: `:param baseplate.events.queue.FieldKind kind: blah blah`

right now the docs render as

![screenshot from 2016-06-17 15 21 09](https://cloud.githubusercontent.com/assets/338853/16166318/28cf3080-349f-11e6-9dc0-620a914dee12.png)